<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="cw">
   <widget class="QWidget" name="wgt_head" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#wgt_head {
	background-color: #f0f0f0;
}</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vl_head">
      <item>
       <widget class="QLabel" name="lb_head">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="wgt_navi_search" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>751</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>750</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#wgt_navi_search {
	background-color: #f0f0f0;
}</string>
    </property>
    <widget class="QWidget" name="wgt_search_kind" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <widget class="QComboBox" name="comb_search_kind">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>0</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <item>
       <property name="text">
        <string>General</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fulltext</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="wgt_exact_matching" native="true">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>5</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <widget class="QCheckBox" name="cb_exact_matching">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Exact Matching</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="wgt_singsongzepe" native="true">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>5</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
     <widget class="QLabel" name="lb_singsongzepe">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="wgt_bar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>51</width>
      <height>451</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>450</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#wgt_bar {
	background-color: #f0f0f0;
}</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>51</width>
       <height>476</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vl_bar">
      <property name="spacing">
       <number>80</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_bar_search">
        <property name="minimumSize">
         <size>
          <width>44</width>
          <height>44</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/:/res/image/search.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_bar_history">
        <property name="minimumSize">
         <size>
          <width>44</width>
          <height>44</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/:/res/image/download.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_bar_setting">
        <property name="minimumSize">
         <size>
          <width>44</width>
          <height>44</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/:/res/image/setting.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_bar_todo">
        <property name="minimumSize">
         <size>
          <width>44</width>
          <height>44</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/:/res/image/todo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="wgt_main_search" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>751</width>
      <height>471</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>750</width>
      <height>470</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#wgt_main_search {
	background-color: #f8f8f8;
}</string>
    </property>
    <widget class="QWidget" name="wgt_search" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>750</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="hl_search">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <item>
        <widget class="QLineEdit" name="le_search">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#le_search {
	border-radius: 10px;
	padding-left: 20px;
	padding-right: 20px;
}</string>
         </property>
         <property name="text">
          <string>Hello</string>
         </property>
         <property name="placeholderText">
          <string>Search books</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_search">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/:/res/image/search_gray.png</normaloff>:/:/res/image/search_gray.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="wgt_books" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>751</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#wgt_books {
	background-color: #ADD8E6;
    border-top-left-radius: 15px; 
    border-top-right-radius: 15px; 
	margin: 2px;
}</string>
     </property>
     <widget class="QScrollArea" name="sa_books">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>731</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ADD8E6;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="sawc_books">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>731</width>
         <height>291</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#scwc_books {
	background-color: #ADD8E6;
    border-top-left-radius: 15px; 
    border-top-right-radius: 15px; 
}</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="wgt_main_singlebookview" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>751</width>
      <height>471</height>
     </rect>
    </property>
    <widget class="QScrollArea" name="sa_singlebookview">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>751</width>
       <height>471</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="sawc_singlebookview">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>749</width>
        <height>469</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="wgt_navi_singlebookview" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>751</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>750</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#wgt_navi_search {
	background-color: #f0f0f0;
}</string>
    </property>
    <widget class="QWidget" name="wgt_singsongzepe_2" native="true">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>5</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
     <widget class="QLabel" name="lb_singsongzepe_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="wgt_navi_history" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#wgt_navi_history {
	background-color: #f0f0f0;
}</string>
    </property>
    <widget class="QWidget" name="wgt_navi_history_search" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#wgt_navi_history_search {
	background-color: #41DBEF;
}</string>
     </property>
     <widget class="QLabel" name="lb_navi_history_search_icon">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="image.qrc">:/:/res/image/search_gray.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lb_navi_history_search">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="wgt_navi_history_download" native="true">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#wgt_navi_history_download {
	background-color: #41DBEF;
}</string>
     </property>
     <widget class="QLabel" name="lb_navi_history_download_icon">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="image.qrc">:/:/res/image/download_gray.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lb_navi_history_download">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="wgt_main_history" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>751</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#wgt_main_history {
	background-color: #f8f8f8;
}</string>
    </property>
    <widget class="QWidget" name="wgt_sub_history_search" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>751</width>
       <height>471</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#wgt_sub_history_search {
	border: 1px solid #41DBEF;
}</string>
     </property>
     <widget class="QScrollArea" name="sa_sub_history_search">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>741</width>
        <height>461</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="sawc_sub_history_search">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>741</width>
         <height>461</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="wgt_sub_history_download" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>751</width>
       <height>471</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#wgt_sub_history_download {
	border: 1px solid #41DBEF;
}</string>
     </property>
     <widget class="QScrollArea" name="sa_sub_history_download">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>741</width>
        <height>461</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="sawc_sub_history_download">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>741</width>
         <height>461</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <zorder>wgt_sub_history_download</zorder>
    <zorder>wgt_sub_history_search</zorder>
   </widget>
   <widget class="QWidget" name="wgt_main_setting" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>751</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="wgt_navi_setting" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>751</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="wgt_main_todo" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>751</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="wgt_navi_todo" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>wgt_main_todo</zorder>
   <zorder>wgt_main_singlebookview</zorder>
   <zorder>wgt_head</zorder>
   <zorder>wgt_navi_search</zorder>
   <zorder>wgt_bar</zorder>
   <zorder>wgt_main_search</zorder>
   <zorder>wgt_navi_singlebookview</zorder>
   <zorder>wgt_main_setting</zorder>
   <zorder>wgt_navi_setting</zorder>
   <zorder>wgt_navi_todo</zorder>
   <zorder>wgt_navi_history</zorder>
   <zorder>wgt_main_history</zorder>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
